packages/firebase/index.d.ts

```diff
@@ -5472,6 +5472,52 @@ declare namespace firebase.storage {
      *     including if the object did not exist.
      */
     updateMetadata(metadata: firebase.storage.SettableMetadata): Promise<any>;
+    /**
+     * List items and folders (prefixes) within this directory.
```

Comment:
Please wrap this to the prevalent line length in the file. I would also add an empty line between these paragraphs.